README.md

```diff
@@ -97,6 +97,7 @@ Service Labels
 
 | Label  | Effect  |
 | ------ | ------- |
+| `k8ify.imagePullSecret: $env_var` | Creates an ImagePullSecret from the referenced environment variable. Can be used once per service|
```

Comment:
Would be nice to have a JSON example here which shows what env_var needs to contain. Adding a code block here may not be great layout-wise but better than leaving the user guessing how the value needs to look.

Reply:
I have added a link to the test spec with an example of what the value looks like.